packages/babel-preset-gatsby-package/__tests__/index.js

```diff
@@ -148,3 +148,51 @@ it(`Specifies proper presets and plugins in debug browser mode`, () => {
     ),
   ])
 })
+
+it(`Specifies proper presets and plugins in production browser mode`, () => {
+  const currentBabelEnv = process.env.BABEL_ENV
+  let presets
+  let plugins
+
+  try {
+    process.env.BABEL_ENV = `production`
```

Comment:
Interesting way to write this!

Why would this ever fail?

Reply:
I dont think setting the BABEL_ENV could actually fail (moved this out of the try-finally now). The try-finally itself is just a guard so that if the `preset()` call would throw, we dont end up running subsequent tests with `BABEL_ENV = 'production'` 🙂

Reply:
Ah - yeah, so I'd recommend just doing something like:

```
let env
beforeEach(() => {
  env = process.env.BABEL_ENV
  process.env.BABEL_ENV = 'production'
})

afterAll(() => {
  process.env.BABEL_ENV = env
})
```